templates/_navbar.html

```diff
@@ -0,0 +1,21 @@
+{% load static %}
+<nav class="navbar navbar-expand-lg navbar-dark custom-navbar">
+    <div class="container-fluid">
+      <a class="navbar-brand" href="{% url 'home' %}">
```

Comment:
```suggestion
      <a class="navbar-brand" href="{% url 'admin_index_page' %}">
```
hiện tại chưa có các page khác nên tạm để thẳng về admin nhé